src/Avalonia.Controls/TextBox.cs

```diff
@@ -416,7 +424,7 @@ protected override void OnKeyDown(KeyEventArgs e)
             }
         }
 
-        protected override void OnPointerPressed(PointerPressedEventArgs e)
+	    protected override void OnPointerPressed(PointerPressedEventArgs e)
```

Comment:
Sure, I have checked before commiting it seemed all space. Something must have gone wrong. Fixing now.